packages/playground/src/components/samples-dropdown.tsx

```diff
@@ -9,6 +9,17 @@ export const SamplesDropdown: FunctionComponent<SamplesDropdownProps> = ({ onSel
     return <option key={sample}>{sample}</option>;
   });
 
+  useEffect(() => {
+    if (window.location.search.length > 0) {
+      const parsed = new URLSearchParams(window.location.search);
+      const sample = parsed.get("sample");
```

Comment:
works great but as talked offline I think an improvement to this would be to be more flexible with the key so instead of requireing `Rest framework` to be passed either of those options should select the rest sample:
- `rest-framework`
- `Rest-Framework`
- `rest framework`
- `ReST-fRAMWORK`